packages/pipelines/package.json

```diff
@@ -33,17 +33,19 @@
     "url": "https://github.com/redhat-developer/aonic-ui/issues"
   },
   "homepage": "https://github.com/redhat-developer/aonic-ui#readme",
+  "dependencies": {
+    "@patternfly/react-core": "^5.1.2",
+    "@patternfly/react-icons": "^5.1.2",
+    "@patternfly/react-table": "^5.1.2",
+    "@patternfly/react-tokens": "^5.1.2"
```

Comment:
Shouldn't these come from peer dependencies?

Reply:
Also, we don't need `"@testing-library/jest-dom": "^6.2.0"` in peer dependencies.

Reply:
Updated.